src/combat.ts

```diff
@@ -9,6 +9,10 @@ export default class Macro extends StrictMacro {
     const macroHead = this.trySkill($skill`Curse of Weaksauce`)
       .trySkill($skill`Micrometeorite`)
       .trySkill($skill`Sing Along`)
+      // eslint-disable-next-line libram/verify-constants
```

Comment:
I don't think the `eslint-disable`s are necessary. The eslint plugin seems to already recognize the skills and item

Reply:
They don't locally; I need to yarn update; but doing so is liable to change the lock file. So I didn't.